source/README_SPHINX.md

```diff
@@ -0,0 +1,99 @@
+# Kokkos Documentation Workflow <br/><br/>
+
+The kokkos-core-wiki is the canonical source of Kokkos documentation "Truth".  Please do not directly edit the main Kokkos project Wiki.<br/>
+
+Sandia employees: Creation of wholly new pages will require Review and Approval (R&A).  Check with your Sandia team lead if you have questions about the R&A process.<br/><br/>
+
+* Step 1) Fork the kokkos-core-wiki repo (`https://github.com/kokkos/kokkos-core-wiki`) if you do not yet have your own fork.<br/>
+
+* Step 2) Clone your fork and add upstream `kokkos-core-wiki` repo as a remote (`git clone git@github.com:ajpowelsnl/kokkos-core-wiki.git`, `git remote add upstream git@github.com:kokkos/kokkos-core-wiki.git`) if you have not already done so. If you already have a fork of the `kokkos-core-wiki`, then simply update your fork and checkout the branch we'll use for transitioning the documentation (`git fetch upstream -p`, `git checkout upstream/sphinx_porting_stage1`).<br/>
```

Comment:
I would highlight the branch name more since, at least for the time being, all new PRs should be posted against that